add redislite to alternatives (#276)
jamesls/fakeredis: README.rst

Alternatives
============

Consider using birdisle_ or redislite_ instead of fakeredis. They embed the redis codebase

> into Python, so it implements the full redis command set and
> behaves far more closely to a real redis implementation.

Reviewer: ```suggestion
into Python, so they implement the full redis command set and
behave far more similarly to a real redis implementation.
```
